velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -23,12 +23,12 @@ namespace facebook::velox::substrait {
 
 /// This class is used to convert Substrait representations to Velox
 /// expressions.
-class SubstraitVeloxExprConverter {
+class SubstraitToVeloxExprConvertor {
```

Comment:
This PR is very large. Some parts of it are just renaming SubstraitVeloxExprConverter to SubstraitToVeloxExprConvertor. Would it be possible to do this renaming in a separate PR to help reduce the size of the PRs and make reviews easier?

Reply:
done